### PR TITLE
Update platform configuration

### PR DESCRIPTION
**Pull request by phieronymus**
No ticket linked to this PR, fixing the types to fix the platform creation in the tests.

### CODE REVIEW

File: src/resources/common/Platform.ts [hunk 1]
```diff
@@ -12,7 +12,20 @@ type CheckoutConfiguration = {
...
+export enum TransferMatchAmount {
+    MAXIMUM = "maximum",
+    MINIMUM = "minimum",
+    EXACT = "exact",
+}
+
+export type BankTransferConfiguration = {
+    enabled: boolean;
+    matchAmount: TransferMatchAmount;
+    expiration: string;
```
Comment (laxa88) [inline src/resources/common/Platform.ts:24]: hmm... i'd call this `expireDate` to better describe the `string`, but not a big deal 😄

Comment (phieronymus) [reply src/resources/common/Platform.ts:24]: Let me double check the type here 🙂

Comment (phieronymus) [reply src/resources/common/Platform.ts:24]: Got the fixture wrong. It seems to be the expiration period. Somehow...

Comment (laxa88) [reply src/resources/common/Platform.ts:24]: oh nice! thanks for the indirect fix XD

**Review by laxa88 — APPROVED** [review]

**Pull request merged**